test/helpers/client.go

```diff
@@ -0,0 +1,25 @@
+package helpers
+
+import (
+	"os"
+
+	"github.com/apache/cloudstack-go/v2/cloudstack"
+	"github.com/pkg/errors"
+	"gopkg.in/ini.v1"
+	"sigs.k8s.io/cluster-api-provider-cloudstack/pkg/cloud"
+)
+
+func NewCSClient() (*cloudstack.CloudStackClient, error) {
+	projDir := os.Getenv("PROJECT_DIR")
+	conf := cloud.Config{}
+	ccPath := projDir + "/cloud-config"
+	if rawCfg, err := ini.Load(ccPath); err != nil {
+		return nil, errors.Wrapf(err, "reading config at path %s:", ccPath)
+	} else if g := rawCfg.Section("Global"); len(g.Keys()) == 0 {
```

Comment:
can you use a constant for Global, or not use Global at all? We're taking a dependency on it even as we're planning to move away from using it. With a constant we'll at least know where it's being used when we're migrating. See https://github.com/aws/eks-anywhere/pull/2399/files for what the new schema will look like and how ini sections will be used as failure domain names

Reply:
Max, I'm happy to make this a constant for now.

That said, I think the jury is still out on how to pass through the credentials when it comes to failure domains.

Myself and others think we should be moving forward with k8s secretes and a yaml file for credentials.

An INI requires an additional name field for a failure domain or a availability zone as is being referred to in the EKS-A design.